### DISCUSSION TITLE
Determine commands to support

### DISCUSSION THREAD

**Discussion by swrhim** [swrhim/rust-discord-bot#2] [Ideas]
We should create a list of commands that should be supported. Based on the discord bot documentations, it shouldn't be "/remind" as there's limitations on the apis we get access to. It should most likely be a message to the bot.

Supported commands:
- remind me
- polls

**Comment by tomlagier** [comment]
Just brainstorming:
* !remindme
* simple games? there's some cool CYOA / idle RPG stuff I've seen in other channels (the PAX discord did this for 2021)
* embedding TFT patch notes?
* funny ascii art?
* scheduling helpers (set up game nights)
* polling (pick games)
* get game ranking for people
* random pics of everyone's dogs

**Reply by swrhim** [reply]
I like these ideas. I think for our first version we should avoid any scheduling or reminders since it would require a storage component. I was thinking for hosting, we can get a digitalocean droplet for $5 a month. thoughts?
